front-end/src/components/modal/create/VideoSelect.js

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import '../../../styles/modalStyles.css';
+import '../../../styles/videoSelectStyles.css';
+import { AiFillFolder } from 'react-icons/ai';
+import { filterFiles } from '../../../lib/videoUtils';
+
+export const VideoSelect = ({ movePage, selectedVideo, setSelectedVideo, files, fileTimespans, videoTimespans }) => {
+
+    const [filteredFiles, setFilteredFiles] = useState([]);
+
+    useEffect(() => {
+        setFilteredFiles(filterFiles(selectedVideo, files, fileTimespans))
+    }, [selectedVideo]);
+
+    return (
+        <div className="videoSelectContainer">
+            <h3>Select Video</h3>
+            <div className="videoSelect">
+                <div className="videoContainer">
+                    {videoTimespans.map((videoTimespan, index) => (
+                        <label
+                            key={index}
+                            className={`videoLabel ${selectedVideo === videoTimespan ? 'selected' : ''}`}
+                            htmlFor={`video-${index}`}
+                        >
```

Comment:
Don't leave floating div tags like this, close it on the above line

Reply:
I personally prefer this format since it aligns with how {} are handled